spec/models/fee_calculator_spec.rb

```diff
@@ -19,6 +19,20 @@
       it { is_expected.to eq(400) }
     end
 
+    context 'はじめの30分100円、30分200円の時' do
+      let(:attr) do
+        {
+         'type' => 'basic',
```

Comment:
[rubocop] Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.